### PR TITLE
Fix build issue | refactor encoder types

### PR DESCRIPTION
**Pull request by subhod-i**

### CODE REVIEW

**Review by davidyuk — CHANGES_REQUESTED** [review]
In general, I'm ok to drop the second parameter of `decode`

File: src/account/multiple.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ class _AccountMultiple extends _AccountResolver {
 
   _resolveAccount (account: Account | string = this.selectedAddress): _AccountBase {
     if (typeof account === 'string') {
-      decode(account, 'ak')
+      decode(account as EncodedData)
```
[the remark this answers is not in the document]
Comment (subhod-i) [reply src/account/multiple.ts]: Sure. Let me just fix that.

File: src/utils/crypto.ts [hunk 2]
```diff
@@ -101,8 +101,8 @@ export function hash (input: Data): Buffer {
  * @param {Number} nonce - Round when contract was created
  * @return {String} - Contract address
  */
-export function encodeContractAddress (owner: EncodedData<'ak'>, nonce: number): string {
```
[the remark this answers is not in the document]
Comment (subhod-i) [reply src/utils/crypto.ts:104]: Let me revert the generic type.

**Review by davidyuk — APPROVED** [review]
Additionally, I've removed unused `PrefixMismatchError`

**Comment by davidyuk** [comment]
👆 this is the first time lgtm find something useful, the second argument should be removed there as well, but we shouldn't forget prefixes while converting to TS

**Pull request merged**